docs/resources/unmanaged_user.md

```diff
@@ -3,10 +3,14 @@ subcategory: "User"
 ---
 # Artifactory Unmanaged User Resource
 
-Provides an Artifactory unmanaged user resource. This can be used to create and maintain Artifactory users.
-
+Provides an Artifactory unmanaged user resource. This can be used to create and manage Artifactory users.
```

Comment:
So I know why its called unmanaged, but it feels confusing to me to call something an `unmanaged` resource and in the next sentence say `create and manage` 

Thoughts?

Reply:
I think it's fine.